trustkit/src/main/java/com/datatheorem/android/trustkit/config/TrustKitConfigurationParser.java

```diff
@@ -225,18 +232,60 @@ private static DomainTag readDomain(@NonNull XmlPullParser parser) throws IOExce
 
         // Parse the domain text
         result.hostname = parser.nextText();
+
         return result;
     }
 
     private static class DebugOverridesTag {
+
         boolean overridePins = false;
         Set<Certificate> debugCaCertificates = null;
     }
 
+    private static Set<Certificate> readTrustAnchors(@NonNull Context context, @NonNull XmlPullParser parser) throws CertificateException,
+        IOException, XmlPullParserException {
+        // Parse trust anchor
+        Set<Certificate> trustAnchorsCertificates = new HashSet<>();
+
+        int eventType = parser.next();
+        while (!((eventType == XmlPullParser.END_TAG) && "trust-anchors".equals(parser.getName()))) {
+            // Look for the next certificates tag
+            if ((eventType == XmlPullParser.START_TAG) && "certificates".equals(parser.getName())) {
+
+                // Parse the supplied certificate file
+                String caPathFromUser = parser.getAttributeValue(null, "src").trim();
+
+                caPathFromUser = formatCertPathResourceWhenId(context, caPathFromUser);
+
+                // Parse the path to the certificate bundle for src=@raw - we ignore system or user
+                // as the src
+                if (!TextUtils.isEmpty(caPathFromUser) && !caPathFromUser.equals("user")
+                    && !caPathFromUser.equals("system") && caPathFromUser.startsWith("@raw")) {
```

Comment:
Why we "don't support system" and later on when building trust manager we copy everything from system's keystore by default?

Reply:
I believe we need systems also as Android N in security config uses those to "extend" system certificates. In other words, if you specify just some certificate in trust-anchors it would trust only that CA and not system ones (if I understood it correctly).
By this we either always trust systems + ours, or always trust only ours (I'm unsure what is the implementation at the end)